src/DynamoCore/Graph/Workspaces/WorkspaceModel.cs

```diff
@@ -1137,6 +1137,7 @@ public virtual void Clear()
             ClearUndoRecorder();
             ResetWorkspace();
 
+            packageDependencies.Clear();
```

Comment:
do we need to do anything to the loadedPackage dependencies property?

@scottmitchell  - does this break any of your assumptions?

Reply:
This makes sense to me. I just didn't see that nodes could be removed outside of `RemoveAndDisposeNode()`. Are there additional ways besides `RemoveAndDisposeNode()` and `Clear()` that we should deal with?

I don't think you need to do anything to loadedPackageDependencies here because those are collected on the fly each time they are accessed, based on the current nodes in the workspace.

Reply:
@scottmitchell Exactly, this is all I need